sdks/python/apache_beam/metrics/execution.py

```diff
@@ -122,6 +125,9 @@ def __repr__(self):
     return 'MetricResult(key={}, committed={}, attempted={})'.format(
         self.key, str(self.committed), str(self.attempted))
 
+  def __str__(self):
+    return pprint.pformat(self)
```

Comment:
Does this actually print a nice representation of self?

Reply:
It does, but I think we can just call repr instead of using a separate ppritn representation. I only used that to debug a bit